### ISSUE TITLE
Maybe a bug: inconsistency between containers final PoS cycle not available

### ISSUE THREAD

**Issue by sebastien-forestier** [massalabs/massa#2385]
I'm not sure if this is a bug, but here are the logs of one of our bootstrap nodes, it ultimately ended with a desynchronization.

```
2022-03-07T11:44:48.000955Z  INFO massa_consensus_worker::consensus_worker: Started cycle 156
2022-03-07T11:44:50.206389Z  INFO massa_bootstrap: bootstrapped peer [::ffff:149.28.186.233]:43592
2022-03-07T11:44:55.941383Z  INFO massa_bootstrap: bootstrapped peer [::ffff:65.21.88.221]:52664
2022-03-07T11:44:59.087039Z  INFO massa_bootstrap: bootstrapped peer [::ffff:157.245.193.215]:40144
2022-03-07T11:44:59.502197Z  INFO massa_bootstrap: bootstrapped peer [::ffff:65.21.254.240]:36122
2022-03-07T11:45:01.403473Z  INFO massa_bootstrap: bootstrapped peer [::ffff:136.244.110.125]:47360
2022-03-07T11:45:04.665998Z  INFO massa_bootstrap: bootstrapped peer [::ffff:109.107.190.5]:50786
2022-03-07T11:45:08.486493Z  INFO massa_bootstrap: bootstrapped peer [::ffff:161.97.172.233]:46972
2022-03-07T11:45:12.828654Z  INFO massa_bootstrap: bootstrapped peer [::ffff:146.19.230.45]:53018
2022-03-07T11:45:16.020761Z  INFO massa_bootstrap: bootstrapped peer [::ffff:94.250.203.111]:60870
2022-03-07T11:45:16.638067Z  INFO massa_bootstrap: bootstrapped peer [::ffff:207.180.197.200]:52804
2022-03-07T11:45:23.320100Z  INFO massa_bootstrap: bootstrapped peer [::ffff:192.241.134.122]:36754
2022-03-07T11:45:24.973027Z  INFO massa_bootstrap: bootstrapped peer [::ffff:147.75.79.127]:37618
2022-03-07T11:45:27.099610Z  INFO massa_bootstrap: bootstrapped peer [::ffff:167.86.113.162]:52520
2022-03-07T11:45:28.279376Z  INFO massa_bootstrap: bootstrapped peer [::ffff:161.97.115.199]:55876
2022-03-07T11:45:30.395953Z  INFO massa_bootstrap: bootstrapped peer [::ffff:185.190.143.41]:41108
2022-03-07T11:45:37.162814Z  INFO massa_bootstrap: bootstrapped peer [::ffff:62.84.121.177]:60346
2022-03-07T11:45:38.145804Z  INFO massa_bootstrap: bootstrapped peer [::ffff:51.250.5.96]:52656
2022-03-07T11:45:41.992190Z  INFO massa_bootstrap: bootstrapped peer [::ffff:65.108.1.95]:58924
2022-03-07T11:45:44.944380Z  INFO massa_bootstrap: bootstrapped peer [::ffff:139.64.165.135]:59890
2022-03-07T11:45:46.334322Z  INFO massa_bootstrap: bootstrapped peer [::ffff:65.21.237.20]:32910
2022-03-07T11:45:47.773276Z  INFO massa_bootstrap: bootstrapped peer [::ffff:162.55.36.137]:50308
2022-03-07T11:45:51.136128Z  INFO massa_bootstrap: bootstrapped peer [::ffff:194.163.135.144]:53584
2022-03-07T11:45:51.773879Z  INFO massa_bootstrap: bootstrapped peer [::ffff:194.163.151.161]:49932
2022-03-07T11:45:57.745248Z  INFO massa_bootstrap: bootstrapped peer [::ffff:65.21.18.46]:59502
2022-03-07T11:45:57.751133Z  INFO massa_bootstrap: bootstrapped peer [::ffff:51.250.26.60]:59496
2022-03-07T11:46:00.523935Z  INFO massa_bootstrap: bootstrapped peer [::ffff:88.99.80.34]:42888
2022-03-07T11:46:01.771330Z  INFO massa_bootstrap: bootstrapped peer [::ffff:185.92.150.10]:59922
2022-03-07T11:46:03.544874Z  INFO massa_bootstrap: bootstrapped peer [::ffff:199.247.24.36]:55554
2022-03-07T11:46:08.428278Z  INFO massa_bootstrap: bootstrapped peer [::ffff:161.97.173.240]:50946
2022-03-07T11:46:10.208436Z  INFO massa_bootstrap: bootstrapped peer [::ffff:167.71.3.96]:33648
2022-03-07T11:46:11.649976Z  INFO massa_bootstrap: bootstrapped peer [::ffff:46.48.40.188]:59440
2022-03-07T11:46:11.940975Z  INFO massa_bootstrap: bootstrapped peer [::ffff:178.128.243.45]:55634
2022-03-07T11:46:15.308862Z  INFO massa_bootstrap: bootstrapped peer [::ffff:135.181.155.49]:50280
2022-03-07T11:46:18.073029Z  INFO massa_bootstrap: bootstrapped peer [::ffff:49.12.202.189]:40376
2022-03-07T11:46:18.763508Z  INFO massa_bootstrap: bootstrapped peer [::ffff:38.242.201.97]:52152
2022-03-07T11:46:23.900119Z  INFO massa_bootstrap: bootstrapped peer [::ffff:135.181.204.42]:59116
2022-03-07T11:46:24.937831Z  INFO massa_bootstrap: bootstrapped peer [::ffff:194.163.181.10]:54288
2022-03-07T11:46:25.599373Z  WARN massa_graph::block_graph: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
Discarding ops: Invalid("block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156")
2022-03-07T11:46:25.599397Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:25.638033Z  INFO massa_protocol_worker::protocol_worker: Connected to node 6hjhEvndUDrHRXJW3H5W8pFP6gKv1FpbwDFjUL2uKTKHMX9YzZ
2022-03-07T11:46:25.656102Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7TPj5V8sBQ1nTgp2JSkHxPkTeeFwEZ5M752MdQ1njjzBMveLgT
2022-03-07T11:46:25.700668Z  INFO massa_protocol_worker::protocol_worker: Connected to node 85sLrYm3SjEEou6Hqdvs443uc2zq9zBSvL261jArrE15opi73f
2022-03-07T11:46:25.757309Z  INFO massa_protocol_worker::protocol_worker: Connected to node 6Pwd4zQ3MMHp8vDRdwJhBRShD9dB6DTSHbJx1sHK2yLBorV7fC
2022-03-07T11:46:26.178862Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7saEGFsqeDMWPhDhrU5JJtUVmoeHDyzLNMT3Y2tYhi1aotGXdZ
2022-03-07T11:46:27.008677Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:27.267234Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7toGq8FPod5oLvCaj62Ae4oMyTAudhztfrvj9jXwjvpUWw6bdZ
2022-03-07T11:46:27.295259Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:27.447757Z  INFO massa_protocol_worker::protocol_worker: Connected to node 66sBbFpwJ7Kh7Qi7tE11d1YJNDcS1PB4amXjWqmwVWv1qMy353
2022-03-07T11:46:27.702486Z  INFO massa_bootstrap: bootstrapped peer [::ffff:207.180.217.124]:57978
2022-03-07T11:46:27.859427Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:28.315291Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7FashTZuDgsdXuLCVEym6UGq2zK2DTGpbYz9ZuY3qQTay82zQJ
2022-03-07T11:46:28.362020Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:28.621333Z  INFO massa_protocol_worker::protocol_worker: Connected to node 8GeEcRFJkxxin1NubUcYoroQrgedeEMYNnmbp69ErqJdFmRNL8
2022-03-07T11:46:28.997283Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:29.815288Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:29.873128Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7bunLxiZX9gkf9EsdyxwChJFfpNrpSM661grrw2TzYWAbUWCsv
2022-03-07T11:46:30.909405Z  INFO massa_bootstrap: bootstrapped peer [::ffff:194.233.83.91]:44012
2022-03-07T11:46:30.930560Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:30.964387Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:30.980903Z  INFO massa_protocol_worker::protocol_worker: Connected to node 6poyu4GHihuy9cbhMNJ2fFEX9WzxdF5eYNgM82kToD74FQcQNb
2022-03-07T11:46:31.458726Z  INFO massa_bootstrap: bootstrapped peer [::ffff:185.187.170.218]:51728
2022-03-07T11:46:31.594631Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:31.623870Z  INFO massa_protocol_worker::protocol_worker: Connected to node 6TdQntBnKuzAfrmfLFXgHbLFQU5x4gdzttBS4uKeJJVBowU7qB
2022-03-07T11:46:32.115047Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:32.176499Z  INFO massa_protocol_worker::protocol_worker: Connected to node 7TefyYcnxUp4A4pj6QCP2RMaZ31SJwFmnhavCHeW3u1ExrWLma
2022-03-07T11:46:32.274590Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:32.294795Z  INFO massa_protocol_worker::protocol_worker: Connected to node 5HwnS9aJQ6Mi8F6i2oDKWk97mdp7Uta9XMUuynmWRWCog4aHNt
2022-03-07T11:46:33.234019Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:33.329547Z  INFO massa_graph::block_graph: consensus.block_graph.maybe_note_attack_attempt DiscardReason::Invalid:discarded because a parent was discarded for the following reason: block graph check_operations error, operation apply to state: there was an inconsistency between containers final PoS cycle not available: 156
2022-03-07T11:46:33.414952Z  INFO massa_protocol_worker::protocol_worker: Connected to node 6p7FKvb2gdhSq2z8NFqBAK55L7EJVEo5vZ3hjUAwi46C2zM2i3
2022-03-07T11:46:33.464071Z  INFO massa_protocol_worker::protocol_worker: Connected to node 8SWbsQEe7ALz
```

**Comment by AureliaDolo** [comment]
It means that at least two cliques were growing in parallel, not allowing finality to settle quick enough.

**Comment by damip** [comment]
corrected in  #2433
